### ISSUE TITLE
⌨ User roles table actions have no discernable text

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#9382]
## ⌨ Accessibility Issue

The buttons to remove and edit a user role on the user page have no discernable text so will not be announced to screen readers.

### WCAG 2.1 Criteria

- https://dequeuniversity.com/rules/axe/4.8/button-name

## 🦋 Expected Behaviour

Button names can be determined by ATs

## 🕵️ Details

This was a simple mistake of using `data-h2-visually-hidden="base(hidden)"` where we should have used `data-h2-visually-hidden="base(invisible)"`.

## 📋 Steps to Reproduce

1. Login as admin
2. Navigate to the edit user page
3. Inspect the delete role button in the role table
4. confirm it has no name in the accessibility tree

## 📸 Screenshot

![Screenshot 2024-02-15 134643](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/a9306a9e-865f-4816-96ec-4522dd3f90d9)

## 🙋‍♀️ Proposed Solution

Simply update the buttons hidden text to use `invisible` instead of `hidden` in the `data-h2-visually-hidden` attribute.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] Text remains visually hidden
- [ ] Text can be determined by the accessibility tree
